### PR TITLE
Registration success

### PR DESCRIPTION
**Pull request by blakepastrana**
Cleaned UI for registration

### CODE REVIEW

File: AgileSignUp/src/main/webapp/WEB-INF/views/attendeeslist.jsp [hunk 1]
```diff
@@ -2,17 +2,33 @@
...
-	<br />
-	<a href="../courses">Return to courses</a>
+	<jsp:include page="adminheader.jsp" /> 	
+	
+	<div class="container">
+		<div class="row">
+			<div class = "col-sm-3 col-centered">
+				<h1 style="text-align: center"> Email List </h1>
+					<c:forEach items="${emailList}" var="email">
+						${email}
```
Comment (lucianojb) [inline AgileSignUp/src/main/webapp/WEB-INF/views/attendeeslist.jsp:23]: Would this generate a list that makes it easy to copy and paste the emails to Outlook?

File: AgileSignUp/src/main/webapp/WEB-INF/views/createuser.jsp [hunk 2]
```diff
@@ -27,32 +27,32 @@
   		<div class= "row">
   			<div class="col-sm-3 col-sm-offset-3">	
   				<div class="form-group">
-					<label for="firstName">First Name</label>
-			 		<sf:input class="form-control" id="firstName" name="firstName" path="firstName" type="text" />
+					<label class="control-label" for="firstName">First Name</label>
+			 		<sf:input class="form-control" id="firstName" name="firstName" path="firstName" type="text" required="true" />
```
Comment (lucianojb) [inline AgileSignUp/src/main/webapp/WEB-INF/views/createuser.jsp:31]: What happens if this field is empty and the register button is pressed?

File: AgileSignUp/src/main/webapp/WEB-INF/views/attendeeslist.jsp (hunk 1 above)
Comment (blakepastrana) [reply AgileSignUp/src/main/webapp/WEB-INF/views/attendeeslist.jsp:23]: Yes, I have tested this. You can highlight+copy then paste in Outlook.

File: AgileSignUp/src/main/webapp/WEB-INF/views/createuser.jsp (hunk 2 above)
Comment (blakepastrana) [reply AgileSignUp/src/main/webapp/WEB-INF/views/createuser.jsp:31]: An elegant alert pops-up stating that the offending form input is required.

**Comment by lucianojb** [comment]
ayyyyyy lgtm

**Pull request merged**